frontend/src/modules/Administration/views/AdministrationView.js

```diff
@@ -13,12 +13,21 @@ import { useState } from 'react';
 import { Helmet } from 'react-helmet-async';
 import { Link as RouterLink } from 'react-router-dom';
 import { ChevronRightIcon, useSettings } from 'design';
-import { AdministrationTeams, DashboardViewer } from '../components';
+import config from '../../../generated/config.json';
+import {
+  AdministrationTeams,
+  DashboardViewer,
+  MaintenanceViewer
+} from '../components';
 
 const tabs = [
   { label: 'Teams', value: 'teams' },
   { label: 'Monitoring', value: 'dashboard' }
 ];
+// Using 'config' as the isModulesEnabled needs Modules.MAINTENANCE which involves much bigger setup which is not needed for maintenance module
```

Comment:
There is a utility in the frontend to check if a module is enabled :) In `frontend/src/utils/helpers/moduleUtils.js` 


You can do something like:
```
import {
  isModuleEnabled,
  ModuleNames
} from 'utils';


if isModuleEnabled(ModuleNames.MAINTENANCE) {
...
}
```